packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -296,3 +371,72 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
     });
   }
 }
+
+class ErrorNavigator extends StatelessWidget {
+  const ErrorNavigator(
+      {Key key,
+      @required this.selectedErrorIndex,
+      @required this.errorCount,
+      @required this.onSelectedErrorIndexChanged})
+      : super(key: key);
+
+  final int selectedErrorIndex;
+  final int errorCount;
+  final void Function(int) onSelectedErrorIndexChanged;
+
+  @override
+  Widget build(BuildContext context) {
+    final label = selectedErrorIndex != null
+        ? 'Error ${selectedErrorIndex + 1}/$errorCount'
+        : 'Errors: $errorCount';
+    return Container(
+      color: devtoolsError,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 6),
+        child: Row(
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(right: 6),
+              child: Text(label),
+            ),
+            IconButton(
```

Comment:
do you need to add `iconSize: defaultIconSize` here?

Reply:
They're quite small using `defaultIconSize` and felt hard to click. What do you think?

`defaultIconSize`:

![Screenshot 2021-02-08 at 18 57 31](https://user-images.githubusercontent.com/1078012/107268065-94640980-6a3f-11eb-8d55-adafa4e7e258.png)

Without:

![Screenshot 2021-02-08 at 18 57 51](https://user-images.githubusercontent.com/1078012/107268070-94fca000-6a3f-11eb-8209-6b7bd2d872a5.png)

Reply:
interesting I agree without looks better